q2_diversity/_alpha/alpha_rarefaction_assets/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "",
+  "version": "",
+  "description": "",
+  "main": "main.js",
+  "dependencies": {
+    "d3": "^4.2.6"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.14.0",
+    "babel-loader": "^6.2.5",
+    "babel-preset-es2015": "^6.14.0",
```

Comment:
These are user-deps as well (basically anything that is needed for the plain webpack compile should be a dep. Linting and HMR are devDependencies.